.simplecov

```diff
@@ -0,0 +1,11 @@
+require 'simplecov-console'
+
+formatters = [SimpleCov::Formatter::HTMLFormatter]
+
+if RSpec.configuration.files_to_run.length > 1
+  formatters << SimpleCov::Formatter::Console
+end
+
+SimpleCov.start 'rails' do
+  formatter SimpleCov::Formatter::MultiFormatter.new(formatters)
+end
```

Comment:
This file configures Simplecov and will generate a helpful coverage table on the console when there's more than one spec being run.